docs/design/2020-06-24-placement-rules-in-sql.md

```diff
@@ -283,41 +314,71 @@ When the `CONSTRAINTS` option doesn't contain `count`, `REPLICAS` must be specif
 
 `REPLICAS` in the statement is equivalent to field `count` in PD placement rule configuration.
 
-### Key range configuration
+#### Key range configuration
 
 In PD placement rule configuration, the key range must be specified. Now that `table_name` is specified in the `ALTER TABLE` statement, key range can be inferred.
 
 Typically, key format is in such a format: `t_{table_id}_r_{pk_value}`, where `pk_value` may be `_tidb_rowid` in some cases. `table_id` can be inferred from `table_name`, thus key range is `t_{table_id}_` to `t_{table_id+1}_`.
 
-Similarly, key range of partitions and indices can also be inferred.
+Similarly, key range of partitions can also be inferred.
 
-### Database placement
+#### Policy Validation
 
-Defining placement rules of databases simplifies the procedures when there are many tables. For example, in a typical multi-tenant scenario, each user has a private database. The dataset in one database is relatively small, and it’s rare to query across databases. In this case, a whole database can be placed in a single region to reduce multi-region latency.
+When placement policies are specified, they should be validated for correctness:
 
-Placement of databases is defined through `ALTER` statements:
+```sql
+CREATE PLACEMENT POLICY `companystandardpolicy` CONSTRAINTS="[+zone=sh]" ROLE=leader REPLICAS=1, CONSTRAINTS="[+zone=sh]" ROLE=follower REPLICAS=1, CONSTRAINTS="[+zone=gz]" ROLE=follower REPLICAS=1;
+ALTER PLACEMENT POLICY `companystandardpolicy` CONSTRAINTS="[+zone=sh]" ROLE=leader REPLICAS=1, CONSTRAINTS="[+zone=sh]" ROLE=follower REPLICAS=1, CONSTRAINTS="[+zone=gz]" ROLE=follower REPLICAS=1;
+CREATE TABLE t1 (
+	id INT NOT NULL PRIMARY KEY,
+	b VARCHAR(100)
+) PLACEMENT POLICY CONSTRAINTS="[+zone=sh]" ROLE=leader REPLICAS=1, CONSTRAINTS="[+zone=sh]" ROLE=follower REPLICAS=1, CONSTRAINTS="[+zone=gz]" ROLE=follower REPLICAS=1;
+ALTER TABLE t1 PLACEMENT CONSTRAINTS="[+zone=sh]" ROLE=leader REPLICAS=1, CONSTRAINTS="[+zone=sh]" ROLE=follower REPLICAS=1, CONSTRAINTS="[+zone=gz]" ROLE=follower REPLICAS=1;
+# Only rule #3 applies for these examples
+CREATE TABLE t1 (a int) PLACEMENT POLICY `mycompanypolicy`;
```

Comment:
I don't get it. Why create `t1` twice?

Reply:
These are different examples, so I will name the table `t2` instead.